.eslintrc

```diff
@@ -1,20 +1,18 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
   "rules": {
-    "quotes": [2, "single"],
-    "strict": [2, "never"],
     "react/jsx-uses-react": 2,
     "react/jsx-uses-vars": 2,
-    "react/react-in-jsx-scope": 2
+    "react/react-in-jsx-scope": 2,
+
+    // Temporary disabled for test/*
+    // with following by https://github.com/eslint/eslint/issues/1416
```

Comment:
Let's change this comment to

"Temporarily disabled for test/* until https://github.com/babel/babel-eslint/issues/33 is resolved"